gdal/doc/source/drivers/raster/gtiff.rst

```diff
@@ -474,6 +476,16 @@ Creation Options
    tile/strip with lossless WebP compression, so you may want to increase the
    BLOCKYSIZE value for strip layout.
 
+-  **JXL_LOSSLESS=YES/NO**: Set whether JPEG-XL compression should be lossless
```

Comment:
same as 👇 For webp we use `TRUE/FALSE` 🤷‍♂️

Reply:
> For webp we use `TRUE/FALSE` man_shrugging

TRUE/FALSE, YES/NO, 1/0, ON/OFF all work for those boolean options